src/lib/flow/include/sol-flow-packet.h

```diff
@@ -122,6 +124,11 @@ struct sol_flow_packet *sol_flow_packet_new_string_take(char *value);
 struct sol_flow_packet *sol_flow_packet_new_blob(const struct sol_blob *value);
 int sol_flow_packet_get_blob(const struct sol_flow_packet *packet, struct sol_blob **value);
 
+struct sol_flow_packet *sol_flow_packet_new_json_object(const struct sol_blob *value);
```

Comment:
would be nice to document in the header that we expect a valid JSON blob, that is, a string fragment with a JSON object (or array in the other packet), it may or not be NULL terminated.